core/src/test/java/bisq/core/util/coin/CoinUtilTest.java

```diff
@@ -88,9 +88,9 @@ public void testGetAdjustedAmount() {
                 20_000_000,
                 1);
         assertEquals(
-                "Minimum allowed trade amount should not be adjusted.",
+                result.toFriendlyString(),
                 "0.01 BTC",
```

Comment:
sorry for the ripple effect here.. hard coded value should be expected and moved up by one line.. same goes to line 103